doc/classes/GPUParticles3D.xml

```diff
@@ -122,6 +126,12 @@
 		</member>
 		<member name="sub_emitter" type="NodePath" setter="set_sub_emitter" getter="get_sub_emitter" default="NodePath(&quot;&quot;)">
 		</member>
+		<member name="trail_enabled" type="bool" setter="set_enable_trail" getter="is_trail_enabled" default="false">
+		</member>
+		<member name="trail_length_secs" type="float" setter="set_trail_length" getter="get_trail_length" default="0.3">
+		</member>
+		<member name="transform_align" type="int" setter="set_transform_align" getter="get_transform_align" enum="GPUParticles3D.TransformAlign" default="0">
```

Comment:
`TransformAlign` is an enum but it hasn't been properly bound as such, this needs to be done in the header with `VARIANT_ENUM_CAST`.

Reply:
Again, `TransformAlign` is bound; so I'm not sure what I'm missing?
https://github.com/godotengine/godot/blob/dea84c9d28efff49054d3ecb2e5f119982cf5845/scene/3d/gpu_particles_3d.h#L181

Reply:
Ah then they need to be bound with `BIND_ENUM_CONSTANT` instead of `BIND_CONSTANT`. @reduz tends to forget one or the other when adding new enums (which happens frequently on big rendering PRs).

Reply:
I've updated the `GPUPartciles3D::TransformAlign` constants to `BIND_ENUM_CONSTANT`, but the static checks are still failing:
```
ERROR: Unresolved enum 'GPUParticles3D.TransformAlign', file: GPUParticles3D
ERROR: Unresolved enum 'GPUParticles3D.TransformAlign', file: GPUParticles3D
ERROR: Unresolved enum 'RibbonTrailMesh.Shape', file: RibbonTrailMesh
ERROR: Unresolved enum 'RibbonTrailMesh.Shape', file: RibbonTrailMesh
```

Reply:
I ran `--doctool .` again and it's all good now. :+1:
Thanks for the help.